src/Tribe/Service_Providers/Context.php

```diff
@@ -371,10 +404,15 @@ static function () {
 					],
 				],
 				'view_request'         => [
-					'read'  => [
+					'read' => [
 						Tribe__Context::WP_MATCHED_QUERY => [ 'eventDisplay' ],
 						Tribe__Context::WP_PARSED        => [ 'eventDisplay' ],
-						Tribe__Context::REQUEST_VAR      => [ 'view', 'tribe_view', 'tribe_event_display', 'eventDisplay' ],
+						Tribe__Context::REQUEST_VAR      => [
+							$view_query_var,
+							'tribe_view',
+							'tribe_event_display',
+							'eventDisplay'
```

Comment:
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma
* :no_entry_sign: Comma required after last value in array declaration

<sub>hash: a5fbd82fec35687fe7ecc1d5683ec0da</sub>